docs/integrations/sources/google-adwords.md

```diff
@@ -60,3 +60,9 @@ When you apply for a token, you need to mention:
 
 If for any reason the request gets denied, let us know and we will be able to unblock you.
 
+
+## Changelog
+
+| Version | Date       | Pull Request | Subject |
+| :------ | :--------  | :-----       | :------ |
+| 0.1.2  | 2021-06-25 | [4205](https://github.com/airbytehq/airbyte/pull/4205) | Set up CDK SAT tests. Incremental tests are disabled due to unsupported state structure in current tests: required structure: {stream_name: cursor_value} given {‘bookmarks’: {stream_name: cursor_value}} |
```

Comment:
```suggestion
| 0.1.2  | 2021-06-25 | [4205](https://github.com/airbytehq/airbyte/pull/4205) | Set up CDK SAT tests. |
```

Reply:
@po3na4skld the log message is wrong, you bump version to 0.2.5, but logs say about 0.1.2.
Also why did you bumped 0.2.3 to 0.2.5, where is 0.2.4?

Reply:
good catch eguene. 

skipping the version isn't a big problem, but we should definitely list the correct version in the changelog